axelar-chains-config/info/testnet.json

```diff
@@ -2453,40 +2318,233 @@
           "owner": "0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05"
         }
       }
-    }
-  },
-  "stellar": {
-    "axelarId": "stellar",
-    "rpc": "https://soroban-testnet.stellar.org",
-    "horizonRpc": "https://horizon-testnet.stellar.org",
-    "networkType": "testnet",
-    "explorer": {
-      "name": "Stellar Expert",
-      "url": "https://stellar.expert/explorer/testnet"
     },
-    "tokenSymbol": "XLM"
-  },
-  "sui": {
-    "name": "Sui",
-    "axelarId": "sui",
-    "networkType": "testnet",
-    "rpc": "https://fullnode.testnet.sui.io:443",
-    "tokenSymbol": "SUI",
-    "chainType": "sui",
-    "explorer": {
-      "name": "Suiscan",
-      "url": "https://suiscan.xyz"
+    "sui": {
+      "name": "Sui",
+      "axelarId": "sui",
+      "networkType": "testnet",
+      "rpc": "https://sui-testnet-rpc.publicnode.com:443",
+      "tokenSymbol": "SUI",
+      "chainType": "sui",
+      "explorer": {
+        "name": "Suiscan",
+        "url": "https://suiscan.xyz/testnet"
+      },
+      "contracts": {
+        "AxelarGateway": {
+          "address": "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be",
+          "versions": [
+            "0x7fa977e8174494413834a2bafd5b46e07049c02d73e771bc00e58a31ef7895be"
+          ],
```

Comment:
make `versions` into a dictionary mapping version number to package id, makes it more obvious instead of relying on the implicit version index